tests/serializer_registry/test_registry.py

```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+
+from bulbs.infographics import data_serializers
+from bulbs.infographics.enum import InfographicType
+from bulbs.serializer_registry import registry
+
+
+
+
+class RegistryTestCase(TestCase):
```

Comment:
I'd add a test case that creates a new serializer (inside the test def) w/ decorator, independent of Infographics